src/partials/px-insights.handlebars

```diff
@@ -0,0 +1,42 @@
+{{#> layout title=title description=description}}
+    <div class="container">
+        <div class="row">
+            <div class="col-md-10 col-xs-9">
+                <h1>
+                    {{{title}}}
+                </h1>
+            </div>
+            {{#if image}}
+                <div class="col-md-2 col-xs-3">
+                    <img class="img-responsive" style="margin-top: 20px;" src="../../img/{{image}}"/>
+                </div>
+            {{/if}}
+        </div>
+        <div class="row">
+            {{{mainContent}}}
+        </div>
+    </div>
+    <script type="text/javascript">
+        $( document ).ready(function() {
+            $('.code-tabs li:first-child').addClass('active');
+        });
+        function copySnippetContent(event){
```

Comment:
It's weird that we have to do this to publish our doc. Isn't there a way to centralize it?